frontend/src/config/i18n/de/guidedTour.json

```
@@ -24,20 +24,29 @@
         "plants_layer_intro": "Die Pflanzen-Ebene ist das Herzstück deiner Planung. Sie erlaubt es dir, Pflanzen auf deiner Karte zu setzen und unterstützt dich, das Gesamtwohl deines Garten dabei zu beachten.",
         "plants_layer_select": "Nutze das Ebenen-Panel und klicke jetzt auf die angezeigte Schaltfläche, um zur Pflanzen-Ebene zu wechseln und Zugriff auf deren Funktionen zu bekommen.",
         "plants_layer_right_toolbar": "Nach dem Wechsel der aktiven Ebene werden automatisch deren Funktionen in den Seitenleisten angezeigt. Die Seitenleiste der Pflanzen-Ebene lässt dich eine Pflanze entweder aus deinen verfügbaren Samen oder durch die Suche auswählen.",
-        "plants_layer_first_planting_search": "Mit diesem Icon kannst du das Suchfeld einblenden lassen, um nach einer beliebigen Pflanze in der umfangreichen PermaplanT-Datenbank zu suchen. Klicke jetzt darauf, um deine erste Pflanze zu suchen.",
-        "plants_layer_first_planting_select": "Suche jetzt nach einer Tomate und klicke sie an, um in den Platzierungsmodus zu wechseln. Keine Sorge, die Aktionen werden im Laufe dieser Einführung rückgängig gemacht werden.",
+        "plants_layer_first_planting_search_text": "Mit diesem Icon kannst du das Suchfeld einblenden lassen, um nach einer beliebigen Pflanze in der umfangreichen PermaplanT-Datenbank zu suchen.",
+        "plants_layer_first_planting_search_action": "Klicke jetzt darauf, um deine erste Pflanze zu suchen.",
+        "plants_layer_first_planting_select_text": "Du wirst nun deine erste Pflanze suchen und platzieren. Keine Sorge, die Aktionen werden im Laufe dieser Einführung rückgängig gemacht werden.",
+        "plants_layer_first_planting_select_action": "Suche jetzt nach einer Tomate und klicke sie an, um in den Platzierungsmodus zu wechseln.",

```

Put this text field higher (where the search text should be written). Please check if quotes work.

```suggestion
        "plants_layer_first_planting_select_action": "Tippe jetzt den Suchbegriff \"Tomate\" ein. Wähle dann per Mausklick die Tomate aus, um in den Platzierungsmodus zu wechseln.",

```